content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,97 @@
+# Capacity Planning
+
+
+Even the most minimally functional `rippled` server must contain the most recently validated Ledger versions to submit transactions to the network and verify the integrity of the XRP Ledger. Beyond these, consider the following server requirements:
+
+- Handling ever-increasing transaction volume
+- Servicing transaction reporting information to clients
+- Maintaining varying amounts of historical data
+
+Capacity planning for `rippled` involves two factors:
+
+- The [configuration settings](#configuration-settings) that affect resource utilization
+- The [network and hardware] (#network and hardware) requirements to achieve consistent, good performance across the XRP Ledger network
+
+## Configuration Settings
+
+Ripple recommends the following guidelines to improve performance. You can set the following parameters in the `rippled.cfg` file to improve performance for your `rippled` server.
+
+### Node Size
+
+The `node_size` parameter determines the size of database caches. Ripple recommends you always set it to `huge` for production servers. Increased cache size requires less disk I/O and allows `rippled` to improve performance. The trade-off to improving performance better is that memory requirements increase.
```

Comment:
> The trade-off to improving performance better

The "better" feels superfluous here. Maybe rephrase the whole paragraph and add a table:

> The `node_size` parameter determines the size of database caches. Larger database caches decrease disk I/O requirements at a cost of higher memory requirements. Ripple recommends you always use the largest database cache your available memory can support. For production servers with the recommended 32 GB of RAM, set `node_size` to `huge`. For development servers, see the following table:
> 
> | Available RAM for `rippled` | Recommended `node_size` value | Notes |
> |---|---|---|
> | < 8 GB | `tiny` | *Not recommended!* |
> | 8 GB | `low` | |
> | 16 GB | `medium` | |
> | 32 GB | `huge` | **Recommended for production servers.** |